eventhubs/data-plane/azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/BaseLease.java

```diff
@@ -126,6 +128,19 @@ public int compareTo(BaseLease other) {
         return this.partitionId.compareTo(other.getPartitionId());
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
@JonathanGiles Is there any guidance around whether to always use `{ }` for single statements or would this be fine?

Reply:
I think we always have braces.
Also, Shawn, this is not the equals method we discussed in our call yesterday (when we used IntelliJ to generate the equals and hashCode methods) - it is totally wrong to check for equality like this.

Reply:
will add braces. Wired, it doesn't has error produced for this.